presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestCorrelationAggregation.java

```diff
@@ -76,6 +77,11 @@ private void testNonTrivialAggregation(double[] y, double[] x)
         PearsonsCorrelation corr = new PearsonsCorrelation();
```

Comment:
It looks like this helper is only used in one place, so I would just change it to take Double[] instead. Also, you could just inline the whole helper